### PR TITLE
Remove requirements for usable items to be in backpack

### LINKED ISSUE #7035: AtragMX, Kestrel, Rangecard not working from backpack
**Arma 3 Version:** `1.92` (stable)
**CBA Version:** `3.11.2` (stable)
**ACE3 Version:** `3.12.6` (stable)

**Mods:**
```
- CBA_A3
- ace
```

**Description:**
AtragMX, Kestrel and Rangecard cannot be opened when they are held in the backpack. Other ACE gadgets I tested (wirecutter, mortar rangetable, **MicroDAGR**, entrenching tool, explosives w/ military clackers, defusal kit, cable ties) are not affected.
They don't open when the hotkey (both _open_ and _show_, in the case of the Kestrel) is pressed, and are not available in the self interaction menu.
Moving them to the vest fixes this, moving them back to the backpack removes them from the self interaction menu and disables the opening/showing hotkeys. I did not test leaving them while moving them to the backpack.

**Steps to reproduce:**
- Carry AtragMX, Kestrel and Rangecard in your backpack
- Press hotkeys to open these menus, nothing happens
- Open self interaction menu, equipment category: the gadgets are not listed there

**Where did the issue occur?**
- Dedicated / Singleplayer / Virtual Arsenal

**Additional information:**
[This file](https://github.com/acemod/ACE3/blob/master/addons/atragmx/initKeybinds.sqf) is not used in the MicroDAGR addon. The "Add deviceKey entry" code for the MicroDAGR is in XEH_clientInit.sqf instead, and there doesn't seem to be any call to `CBA_fnc_addKeybind`.

**RPT log file:**
https://gist.github.com/Blutze/0c1cf00ceba8dea127a9cbc05070339a

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Close #7035

### CODE REVIEW

File: addons/atragmx/functions/fnc_can_show.sqf [hunk 1]
```diff
@@ -15,4 +15,5 @@
  * Public: No
  */
 
-(("ACE_ATragMX" in (uniformItems ACE_player)) || ("ACE_ATragMX" in (vestItems ACE_player))) && !(underwater ACE_player);
+!underwater ACE_player &&
+"ACE_ATragMX" in ([ACE_player] call CBA_fnc_uniqueUnitItems)
```
Comment (PabstMirror) [inline addons/atragmx/functions/fnc_can_show.sqf]:
```suggestion
{"ACE_ATragMX" in ([ACE_player] call CBA_fnc_uniqueUnitItems)}
```

**Comment by jonpas** [comment]
What about https://github.com/acemod/ACE3/blob/1fe7c5c29c3e5315c33ba4547ec8495c7cc2e3b8/addons/map/functions/fnc_getUnitFlashlights.sqf#L36?

**Comment by mharis001** [comment]
That also checks weapon items so it needs to use `CBA_fnc_uniqueUnitItems`.

**Comment by DJ-Reticuli** [comment]
Any restrictions on backpack items I want gone from my Ace.  How to do it?

**Pull request merged**